vnext/Mso/dispatchQueue/readMe.md

```diff
@@ -0,0 +1,70 @@
+# dispatchQueue liblet
```

Comment:
>dispatchQueue [](start = 2, length = 13)

nit: this file is conventionally README with all caps

Reply:
same for future

---
In reply to: [368202671](https://github.com/microsoft/react-native-windows/pull/3905#discussion_r368202671) [](ancestors = 368202671)